core/common/src/main/java/alluxio/Constants.java

```diff
@@ -107,7 +107,6 @@
   public static final String LINEAGE_MASTER_CLIENT_SERVICE_NAME = "LineageMasterClient";
   public static final String META_MASTER_CLIENT_SERVICE_NAME = "MetaMasterClient";
   public static final String META_MASTER_MASTER_SERVICE_NAME = "MetaMasterMaster";
-  public static final String META_MASTER_SERVICE_NAME = "MetaMaster";
```

Comment:
The string value for `META_MASTER_CLIENT_SERVICE_NAME` needs to be "MetaMaster" so that 1.7 clients can talk to 1.8 meta master

Reply:
fixed and add a TODO